src/ClientWidgetApi.ts

```diff
@@ -511,17 +522,17 @@ export class ClientWidgetApi extends EventEmitter {
                 });
             }
 
-            if (request.data.future_timeout === undefined && request.data.future_group_id === undefined) {
+            if (!isDelayedEvent) {
                 sendEventPromise = this.driver.sendEvent(
                     request.data.type,
                     content,
                     null, // not sending a state event
                     request.data.room_id,
                 );
             } else {
-                sendEventPromise = this.driver.sendFuture(
-                    request.data.future_timeout ?? null,
-                    request.data.future_group_id ?? null,
+                sendEventPromise = this.driver.sendDelayedEvent(
+                    request.data.delay ?? null,
+                    request.data.parent_delay_id ?? null,
```

Comment:
In the rust sdk I now decided to not include the parent delay id. Do you think it makes sense if I add it back in or is it fine to have this different as long as there is no spec for the parent concept?

Reply:
IMO since nothing actually uses the parent ID yet, it's fine for some components to be missing the parent ID, as long as its absence doesn't break anything (which it shouldn't, since it's a superset of having just the delay parameter).